class/Heavyarms; Gunslinger.json

```diff
@@ -0,0 +1,2882 @@
+			]
+		}
+		
+	],
+	"feat": [
+		{
+			"name": "Bully",
+			"source": "HA:GS",
+			"prerequisite": [
+				{
+					"other": "Proficiency with at least one firearm"
+				}
+			],
+			"entries": [
+				"You are adept at using firearms in close quarters, gaining the following benefits:",
+				{
+					"type": "list",
+					"items": [
+						"Being within 5 feet of a hostile creature doesn't impose disadvantage on your ranged attack rolls.",
+						"Once per turn, when you hit a creature within 10 feet of you with a ranged {@action attack} using a firearm, you can force the target to make a Strength saving throw (DC equal to 8 + your proficiency bonus + your Dexterity modifier). On a failed save, you push the target 5 feet away from you and knock it {@condition prone}."
+					]
+				}
+			]
+		},
+		{
+			"name": "Marksman",
+			"source": "HA:GS",
+			"entries": [
+				"You are an expert at hitting your mark with ranged weapons, gaining the following benefits:",
+				{
+					"type": "list",
+					"items": [
+						"Being prone doesn't impose disadvantage on your ranged weapon attacks.",
+						"Attacking a prone target doesn't impose disadvantage on your ranged weapon attack rolls if the target is within your weapon's normal range.",
+						"If you haven't moved since the start of your turn, you can use a bonus action to steady your aim. Until the end of your next turn, you have advantage on the next ranged weapon attack roll you make. This effect ends early if you move or take damage."
+					]
+				}
+			]
+		},
+		{
+			"name": "Pistolero",
+			"source": "HA:GS",
+			"prerequisite": [
+				{
+					"other": "Proficiency with pistols"
+				}
+			],
+			"entries": [
+				"Your speed and skill with a six-gun is unmatched. You gain the following benefits:",
+				{
+					"type": "list",
+					"items": [
+						"When you take the Attack action and attack with a one-handed weapon, you can use a bonus action to attack with a pistol you are holding in your other hand.",
+						"If you are surprised at the beginning of combat and aren't incapacitated, you can act normally on your first turn, but only if you attack with a pistol before doing anything else on that turn."
+					]
+				}
+			]
+		}
+	],
+	"table": [
+		{
+			"name": "Combat Analysis: Damage Per Round",
+			"source": "HA:GS",
+			"page": 0,
+			"type": "tableGroup",
+			"tables": [
+				{
+					"name": "",
+					"source": "",
+					"type": "table",
+					"intro": [
+						"The following table presents the average damage of the gunslinger based on a Monte Carlo method simulating 100,000 rounds of attacks at each level against a Normal Distribution of Armor Class values appropriate for that level. Similar data sets using existing builds are also presented following the same methodology for comparative purposes."
+					],
+					"colLabels": [
+						"",

```

Comment:
should be strings, then use `text-center` in `colStyles`

Reply:
Ah so the exit code is default for whatever

Reply:
I remember this part, Lyra actually wrote it. It functions fine but I remember him saying you might not approve it lol

Reply:
and he was correct
any reason my proposed fix doesn't work?

Reply:
finished the changes